serialization/src/main/scala/sbt/serialization/CanToString.scala

```diff
@@ -23,6 +35,17 @@ object CanToString {
     _.toString, {
       s: String => TypeExpression.parse(s)._1
     })
```

Comment:
these three seem kind of BS here - they should really be wherever we have the file/uri/typeexpression pickler